packages/vue3-vite/src/generators/library/files/postcss.config.js.template

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: []
+}
```

Comment:
I'm not sure what should I put in `postcss.config.js.template`, so currently the only property is `plugins: []`.

Reply:
Since we include "postcss-basics", it could be installed here. But, I'm not sure it's a good idea to force that library on everyone who uses `nx-vue3-vite` 😅 

I originally included that and my fetch wrapper library because I'm using them in all my projects, but for this project to be useful to more people it might be better to remove those two.